.gitignore

```diff
@@ -79,3 +79,5 @@ dmypy.json
 
 # Virtual environment
 venv/
+
+examples/caching_and_replay/my_cache_file_path.cache
```

Comment:
This is the default file name in the `caching_and_replay` example, so I suppose it is fine to be added in .gitignore.